### PR TITLE
#164796708 Built page for viewing user profiles

### PR DESCRIPTION
**Pull request by filschristian**
#### What does this PR do?
Add a page for viewing active user's profiles

#### Description of Task to be completed?
This PR has a page for viewing active user's profiles.

#### How should this be manually tested?
Pull this branch
Launch the server and navigate to the HTML page `profile-list.html`

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
#164796708

#### Screenshots (if appropriate)

- Desktop Version: 
image
- Mobile Version: 
image

#### Questions:

### CODE REVIEW

**Review by NkFab — CHANGES_REQUESTED** [review]
@filschristian  I think it would look better if the text cuts on the vertical line where the follow button starts. The follow button should be on the same line horizontally with the Names of the author.

File: ui/css/style.css [hunk 1]
```diff
@@ -1775,3 +1775,47 @@ textarea::placeholder {
...
+  overflow: hidden;
+  text-overflow: ellipsis;
+  line-height: 2.8vh; }
+
+.follow-button {
+  width: 30%;
+  height: 90%;
+  margin-left: auto; }
+
+.follow-button button {
```
Comment (oesukam) [inline ui/css/style.css:2133]: @filschristian Try to use sass nested principle effectively to avoid overwriting other buttons

**Review by oesukam — CHANGES_REQUESTED** [review]
@filschristian the follow button for the mobile version looks much bigger, try to reduce the height to see how it looks

File: ui/css/style.css (hunk 1 above)
Comment (filschristian) [reply ui/css/style.css:2133]: Good feedback implemented, Thanks @oesukam

**Comment by filschristian** [comment]
> @filschristian I think it would look better if the text cuts on the vertical line where the follow button starts. The follow button should be on the same line horizontally with the Names of the author.

Good, feedback implemented, Thanks @NkFab

**Comment by filschristian** [comment]
> @filschristian the follow button for the mobile version looks much bigger, try to reduce the height to see how it looks

No, they don't, they look big on the screenshot only. @oesukam

**Review by NkFab — APPROVED** [review]
Good work @filschristian

**Review by oesukam — APPROVED** [review]

**Pull request merged**